### ISSUE TITLE
Windows build defaults to daemonising & doesn't provide -D switch

### ISSUE THREAD

**Issue by leonsodhi** [papertrail/remote_syslog2#3]
```
C:\remote_syslog>remote_syslog.exe -h
Usage of remote_syslog.exe:
  -c, --configfile="/etc/log_files.yml": Path to config
      --debug-log-cfg="": the debug log file
  -d, --dest-host="": Destination syslog hostname or IP
  -p, --dest-port=0: Destination syslog port
      --eventmachine-tail=false: No action, provided for backwards compatibility

  -f, --facility="user": Facility
      --hostname="": Local hostname to send from
      --log="<root>=INFO": "logging configuration <root>=INFO;first=TRACE"
      --new-file-check-interval={0}: How often to check for new files
      --no-eventmachine-tail=false: No action, provided for backwards compatibility
      --pid-file="": Location of the PID file
  -s, --severity="notice": Severity
      --tcp=false: Connect via TCP (no TLS)
      --tls=false: Connect via TCP with TLS
```

and

```
C:\remote_syslog>remote_syslog.exe -D
unknown shorthand flag: 'D' in -D

C:\remote_syslog>remote_syslog.exe --no-detach=true
unknown flag: --no-detach
```

Running it normally:

```
C:\remote_syslog>remote_syslog.exe
panic: cannot daemonize on windows
```

**Comment by t-oster** [comment]
This seems to be still not fixed in v0.20-beta1

**Comment by stanislavromanov** [comment]
Why is this issue closed? This is still an issue on `v0.20-stable`.
